azure-pipelines/build/doclient-lite-docker.yml

```diff
@@ -14,26 +14,21 @@ schedules:
     include:
     - main
 
-pool: dotestlab
+pool: 1es_hosted_pool_ubuntu_1804
 
 jobs:
 - template: templates/do-docker-jobs.yml
   parameters:
-    targetOS: 'debian10'
+    targetOsArch: 'debian9_arm32'
     targetArch: 'arm32'
```

Comment:
>targetArch: 'arm32' [](http://example.com/codeflow?start=4&length=19)

If you don't go back to separating the OS and arch, remove targetArch here.

Reply:
missed this one